app/assets/stylesheets/content/work_packages/resizer/resizer.sass

```diff
@@ -1,6 +1,7 @@
 .work-packages--resizer
   position: absolute
   top: 50%
+  left: -2px
   cursor: col-resize
   color: grey
```

Comment:
Is it on purpose that the color of the icon is different from the color of the vertical line?

Reply:
Actually yes. The border has the color `#eee`. When using this on the icon it is so bright, that it is really hard to see.